packages/xstate-react/src/types.ts

```diff
@@ -26,25 +26,6 @@ export interface Subscribable<T> {
   ): Subscription;
 }
 
-export interface ActorRef<TEvent extends EventObject, TEmitted = any>
```

Comment:
were those considered to be public API? i don't mind breaking such things when it comes to TS in minor versions but if using those was advertised in issues etc then we should at least mention that they have been removed/moved elsewhere

Reply:
Deleted here because it's available in `XState` - we can just re-export it if we don't want to break things?

Reply:
This would still be a breaking change because `@xstate/react` can be used with versions of `xstate` that do not export those. Unfortunately, peer dependencies are PITA in that regard. I believe that the simplest way around this would be to create `@xstate/types` package that could be a regular dependency of both.

Reply:
> I believe that the simplest way around this would be to create `@xstate/types` package that could be a regular dependency of both.

Not a fan of adding dependencies, especially if these types should ideally live in `xstate`. I think a peer-dependency should be fine.

Reply:
I added the `ActorRef` back, but as the same `ActorRef` as what was added in XState core. That should be all that is necessary to prevent breaking changes.